src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,13 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(n -> Arrays.stream(n.split(",")))
+                .map(s -> Integer.parseInt(s))
```

Comment:
method reference use 
```suggestion
                .map(Integer::parseInt)
```